2016/05.md

```diff
@@ -0,0 +1,67 @@
+<img src="../images/Ecma_RVB-003.jpg" align="right" height="70" alt="" />
+
+## Agenda for the 52nd meeting of Ecma TC39
+
+- **Host**: Google, Munich, Germany
+- **Dates**: *Monday*, 23 May 2016 to *Wednesday*, 25 May 2016
+- **Times**:
+  - 10:00 to 17:00 CEST on 23 and 24 May 2016
+  - 10:00 to 16:00 CEST on 25 May 2016
+- **Location**: [Google Office, Erika-Mann-Str. 33, 80636 München](https://www.google.de/maps/place/Google/@48.1451203,11.5443629,17z/data=!4m2!3m1!1s0x0000000000000000:0xdd1b393a488b0ebf?hl=en)
+- **Wifi**: TBD
+- **Monday Dinner**:
+  - Time: TBD
+  - Location: TBD
+- **Tuesday Dinner**:
+  - Time: TBD
+  - Location: TBD
+- **Contact**:
+  - Name: Andreas Rossberg
+  - Phone: +49 173 300 5690
+  - Email: rossberg@google.com
+
+## Logistics
+
+### Registration
+
+[Doodle](http://doodle.com/poll/kwyxqnc95r9a3f8b)
```

Comment:
@rossberg-chromium normally Patrick will set up the doodle under the ecma-international org, just email them for the link.